lib/core/instance.js

```diff
@@ -176,7 +175,7 @@ var RussInstance = function () {
         var isDel = opts.concurrent || opts.sequence;
         if (opts.name && opts.doc && (hasFunc || isDel)) _this4.tasks[opts.name] = opts;else throw new Error(ERR_MSG);
       };
-      if (files.length === 0) throw new Error('No tasks defined in russ.tasks');
+      if (!files || files.length === 0) throw new Error('No tasks defined in russ.tasks');
```

Comment:
If `tasks` definition is left inside `try/catch` the `file` check within `register` won't be necessary, it will either throw an error because the directory doesn't exist or will throw an error that no tasks are defined because of an empty `files` array. In which case the check could just simply be:

```
if (!files.length) throw new Error('No tasks defined in russ.tasks');
```